roles/deploy/tasks/main.yml

```diff
@@ -0,0 +1,80 @@
+---
+- name: Initialize
+  deploy_helper: path="{{ project_root }}" state=present
+
+- name: Clone project files
+  git: repo="{{ project_git_repo }}"
+       dest="{{ project_source_path }}"
+       version="{{ project_version }}"
+       accept_hostkey=yes
+
+- name: write unfinished file
+  file: path="{{ project_source_path }}/{{ deploy_helper.unfinished_filename }}" state=touch
+
+- name: Copy files to new build dir
+  command: cp -pr {{ project_source_path }} {{ deploy_helper.new_release_path }}
+
+- name: Move site into root
+  shell: mv {{ deploy_helper.new_release_path }}/{{ project_subtree }}/* {{ deploy_helper.new_release_path }}
+  when: project_subtree is defined
```

Comment:
A common setup including one that we use for roots.io is this dir tree:

```
roots.io/
  ansible/
  roots.io/
```

ie: your repo has two top level folders: ansible/playbooks (customized bedrock-ansible), and your actual site/Bedrock project. So we have `project_subtree: roots.io` and it just moves that up a level.